### PR TITLE
[API/RPC] BlockModel Time and MedianTime should be of type long

### PR DESCRIPTION
**Pull request by fassadlr**

### CODE REVIEW

File: src/Stratis.Bitcoin/Controllers/Models/BlockModel.cs [hunk 1]
```diff
@@ -36,12 +34,10 @@ public class BlockModel
         public object[] Transactions { get; private set; }
 
         [JsonProperty("time")]
-        [JsonConverter(typeof(DateTimeOffsetConverter))]
-        public DateTimeOffset Time { get; private set; }
+        public long Time { get; private set; }
 
         [JsonProperty("mediantime")]
-        [JsonConverter(typeof(DateTimeOffsetConverter))]
```
Comment (fenix2222) [inline src/Stratis.Bitcoin/Controllers/Models/BlockModel.cs]: Doesn't `DateTimeOffsetConverter` already convert it to UnixTimeSeconds? Just remove `.ToString()` from converter... I would rather deal with DateTimeOffset in dotnet than with longs

Comment (fassadlr) [reply src/Stratis.Bitcoin/Controllers/Models/BlockModel.cs]: @fenix2222 I don't want to make changes to `DateTimeOffsetConverter` as it is used in numerous other places.

Comment (fenix2222) [reply src/Stratis.Bitcoin/Controllers/Models/BlockModel.cs]: Hmm, it should be consistent, somewhere we use DateTimeOffset, in other places longs. I think the converter was meant to make it all consistent and probably other usages of it should be fixed. Maybe not for this PR though

Comment (fassadlr) [reply src/Stratis.Bitcoin/Controllers/Models/BlockModel.cs]: Whats the issue with longs as opposed to DateTimeOffSet?

Comment (fenix2222) [reply src/Stratis.Bitcoin/Controllers/Models/BlockModel.cs]: 1. Consistency, all other models mainly use DateTimeOffset, so as a consumer of the model I would expect it to be consistent
2. DateTimeOffset is just a much richer object with helper properties and methods and closer describes the property - Time, CreationTime.

**Pull request merged**